action/reward_common.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type reward_common struct{}
+
+func (*reward_common) EthTo() (*common.Address, error) {
```

Comment:
function receiver should be `reward_common`, consistent with usage in actions

Reply:
using pointer receiver is more efficient, it will call the func using caller's pointer/address. assume after compile, the func signature is `XXXEthTo(*reward_common)`
```
var c stake_create
c.EthTo() ==> XXXEthTo(&c.reward_common), here it uses the address of `c.reward_common`
```
if you use struct receiver, the func signature will become `XXXEthTo(reward_common)`
```
var c stake_create
c.EthTo() ==> XXXEthTo(c.reward_common), here it will create a copy of `c.reward_common` struct and pass to func
```